### ISSUE TITLE
Default settings echo test results twice

### ISSUE THREAD

**Issue by Raydiation** [mikeerickson/gulp-phpunit#22]
I'm running version 0.8.1 on Arch Linux

Here's my output (note that it gets written twice)
```
$ gulp test-php
[15:03:52] Using gulpfile ~/programming/core/apps/notes/js/gulpfile.js
[15:03:52] Starting 'test-php'...
[15:03:52] Finished 'test-php' after 6.96 ms
PHPUnit 4.3.5 by Sebastian Bergmann.

Configuration read from /home/bernhard/programming/core/apps/notes/phpunit.xml

..................................

Time: 308 ms, Memory: 16.00Mb

OK (34 tests, 107 assertions)
[15:03:52] PHPUnit 4.3.5 by Sebastian Bergmann.

Configuration read from /home/bernhard/programming/core/apps/notes/phpunit.xml

..................................

Time: 308 ms, Memory: 16.00Mb

OK (34 tests, 107 assertions)
```

This is my build file:
```js
var phpunit = require('gulp-phpunit');

gulp.task('test-php', function () {
    'use strict';
    gulp.src('../phpunit.xml')
        .pipe(phpunit('phpunit'));
});
```

Adding the silent option slims it down to only one output, which however seems weird:
```js
var phpunit = require('gulp-phpunit');

gulp.task('test-php', function () {
    'use strict';
    gulp.src('../phpunit.xml')
        .pipe(phpunit('phpunit', {silent: true}));
});
```

**Comment by mikeerickson** [comment]
@Raydiation I have tried a number of permutations and not one produces the output twice?   Do you have any further information on this issue that you could share to help me figure out what is happening?

**Comment by Raydiation** [comment]
Ok, the issue is gone, don't ask me why.

**Comment by prometheas** [comment]
So, I hate to say it, but I'm getting the same problem with the double output. Interestingly, however, if I add the `silent: true` option, I get the output only once.

**Comment by mikeerickson** [comment]
Would you be willing to share a bit more information?  A rather bizarre issue, but would love to track it down and fix

Any additional info you can share (or a small sample test app) I would be happy to dig deeper.

**Comment by prometheas** [comment]
Definitely. I've posted my deets a few times, but check out #25 for the full picture (task JS, phpunit.xml, and output with debug).

**Comment by mikeerickson** [comment]
This has been fixed in 0.9.0, please update :-)
